_stbt/core.py

```diff
@@ -1422,6 +1423,28 @@ def _mainloop():
               "is still alive!" if thread.isAlive() else "ok"))
 
 
+class _Annotation(namedtuple("_Annotation", "pts region label colour")):
+    MATCHED = (32, 0, 255)  # Red
+    NO_MATCH = (32, 255, 255)  # Yello
+
+    @staticmethod
+    def from_result(result, label=""):
+        colour = _Annotation.MATCHED if result else _Annotation.NO_MATCH
+        return _Annotation(result.timestamp, result.region, label, colour)
+
+    def draw(self, img):
+        if not self.region:
+            return
+        cv2.rectangle(
+            img, (self.region.x, self.region.y),
+            (self.region.right, self.region.bottom), self.colour,
+            thickness=3)
+
+        # Slightly above the match annotation
+        label_loc = (self.region.x, self.region.y - 10)
+        _draw_text(img, self.label, label_loc, (255, 255, 255), font_scale=0.5)
```

Comment:
I'm not concerned about efficiency, but I think it would be a good idea to avoid drawing a black box with no text in if `label == ""`. I'll add that to `_draw_text` though rather than here.